packages/react-native/scripts/react_native_pods.rb

```diff
@@ -139,7 +139,7 @@ def use_react_native! (
   pod 'Yoga', :path => "#{prefix}/ReactCommon/yoga", :modular_headers => true
 
   pod 'DoubleConversion', :podspec => "#{prefix}/third-party-podspecs/DoubleConversion.podspec"
-  pod 'glog', :podspec => "#{prefix}/third-party-podspecs/glog.podspec"
+  pod 'glog', :podspec => "#{prefix}/third-party-podspecs/glog.podspec", :modular_headers => true
```

Comment:
When adding `"DEFINES_MODULE" => "YES"` to `s.pod_target_xcconfig` in `packages/react-native/third-party-podspecs/glog.podspec` I still get the following error:

```
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `MyNativeView` depends upon `glog`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
```

Reply:
yeah, glog is a different beast as it is a c++ library we don't own and that does not actually support Cocoapods. So we are doing some black magic to make it work.